internal/crypto/openpgp/crypter_test.go

```diff
@@ -0,0 +1,61 @@
+package openpgp
+
+import (
+	"bytes"
+	"io/ioutil"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/wal-g/wal-g/internal/crypto"
+	"golang.org/x/crypto/openpgp"
+)
+
+var pgpTestPrivateKey string
+
+const PrivateKeyFilePath = "./testdata/pgpTestPrivateKey"
+
+func init() {
+	pgpTestPrivateKeyBytes, err := ioutil.ReadFile(PrivateKeyFilePath)
+	if err != nil {
+		panic(err)
+	}
+	pgpTestPrivateKey = string(pgpTestPrivateKeyBytes)
+}
+
+func CreateCrypter(armedKeyring string) *Crypter {
```

Comment:
It looks like a reusable function not only in tests. Similar to NewCrypter.

Reply:
Moved from tests to actual package.